### DISCUSSION TITLE
#100daysofgatsby 2021 late starter - need help

### DISCUSSION THREAD

**Discussion by cmilam** [gatsbyjs/gatsby#29099] [Help]
Hi I got into the #100daysofgatsby challenge a little bit late. I've looked through the posts for the challenge and most are just people showing off their website. I got everything installed and configured on my machine and I can access my initial installation on localhost:8000. When I signed up for contentful I used the example API key they provided. I'm not sure what to do next. Do I create content models? 

Also I signed up for gatsby cloud and am unclear how to push my site to gatsby cloud. I get stuck trying to hook it up to my newly created github account. I'm fine at creating sites with HTML and CSS and I know enough javascript to mess things up really bad (so not much at all). But I'm currently stuck with the gatsby index page. I'm not sure how to proceed. Any tips or doc suggestions would be very helpful. Thanks!

**Comment by cmilam** [comment]
Ok so I found this doc and will try this for connecting to github - https://support.gatsbyjs.com/hc/en-us/articles/360059155134-Create-a-Site-from-a-Template

**Reply by cmilam** [reply]
Ok first I tried setting up the blog with contentful option - it created the repo in github but could not create a space in contentful. I went back and just installed gatsby blog. Not sure what the next step is.

**Comment by Aereli** [comment]
It wants you to create content on a CMS and pull it onto your gatsby site. 
I am not familiar with Contentful so I used [sanity.io](https://www.sanity.io/) instead as my headless CMS. 
This is my [project](https://github.com/Aereli/100DaysOfGatsby2021) so you can check out maybe it'll help you.

It's not complete because I need to push it onto the cloud which I'll be doing later tonight.

**Reply by cmilam** [reply]
Thanks. It is actually helpful to look at your set up a little bit. Apparently I've still not installed the right site to [my github](https://github.com/cmilam/). I have a Gatsby starter blog with contentful that didn't hook up, and I have a gatsby starter blog. Looks like I just need the gatsby starter. Will have to figure out which one that is. Will take a look later, but thanks for sharing the link to your github.

**Reply by cmilam** [reply]
Oh man, frustrating. Now I have 4 github projects none of them look like yours. The default starter is different from the minimal starter that you have. I don't know where to find that in the gatsby cloud.

**Reply by cmilam** [reply]
Ok I finally got my github linked to my local repo and it's running correctly

**Reply by Aereli** [reply]
They are currently showing the solution on this LIVE on youtube. I’m sure it'll still be up once the live course ends incase you dont make it in time. https://www.youtube.com/watch?v=rOyE0CTw6Ko
